hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationServiceImpl.java

```diff
@@ -392,7 +392,20 @@ public boolean isCallTimedOut(Operation op) {
     @Override
     public Map<Integer, Object> invokeOnPartitions(String serviceName, OperationFactory operationFactory, int[] partitions)
             throws Exception {
-        return invokeOnPartitions(serviceName, operationFactory, toIntegerList(partitions));
+        // todo: copy paste logic from the above code.
+        Map<Address, List<Integer>> memberPartitions = new HashMap<Address, List<Integer>>(3);
```

Comment:
Let me see what happened here. Perhaps I didn't start from the right branch since this is from another cleanup